cmd/gapit/trace.go

```diff
@@ -49,12 +49,17 @@ func init() {
 	})
 }
 
+const coreAPI = uint32(1 << 0)
```

Comment:
Please add a comment saying these hard-coded constants need to be kept in-sync with the `api_index` in the API files.